ros/src/models/chrono/ros_chrono/src/path_following.cpp

```diff
@@ -200,6 +195,14 @@ int main(int argc, char* argv[]) {
     ros::init(argc, argv, "path_follower");
     ros::NodeHandle node;
 
+    // Rigid terrain dimensions
+    double terrainHeight;
+    double terrainLength;  // size in X direction
+    double terrainWidth;   // size in Y direction
+
+    node.getParam("system/terrain/terrainHeight",terrainHeight);
+    node.getParam("system/terrain/terrainLength",terrainLength);
+    node.getParam("system/terrain/terrainWidth",terrainWidth);
 
     // Declare ROS subscriber to subscribe planner topic
     std::string planner_namespace;
```

Comment:
I think that we can delete all of these

Reply:
No. These parameters are to set up the chrono ground. Previously there was a bug where the chrono vehicle always sinks into the ground, and then continues falling down. This bug is fixed by increasing the terrain size of the ground. The current value of them is 700, and this is now read from the yaml file. The parameter terrainWidth and terrainLength will be called later in this app.